coala_quickstart/generation/FileGlobs.py

```diff
@@ -28,6 +30,12 @@ def get_project_files(log_printer, printer, project_dir):
                       color="green")
         ignore_globs = get_gitignore_glob(project_dir)
 
+    if noninteractive:
+        if ignore_globs is None:
+            file = open(".gitignore", "w")
```

Comment:
You need to create this file in the *project directory*. Not the current directory.

This should be `os.path.join(project_dir, ".gitignore")`, not `".gitignore"`. See line 27 of this file for example.